translations/README/SWAHILI.md

```diff
@@ -337,12 +337,10 @@ Ninaahidi nitajaribu kukagua na kuunganisha haraka iwezekanavyo lakini ninafanya
 - Unaweza **kunifuata** na kuwasiliana na [Twitter](https://twitter.com/Syknapse '@Syknapse') au [kutumia mojawapo ya chaguo hizi nyingine](https://syknapse.github. io/Syk-Houdeib/#contact 'Sehemu yangu ya mawasiliano | Kwingineko')
 - Huu ni mradi wa programu huria kwa hivyo mbali na kuchangia kadi yako, unakaribishwa kukusaidia kurekebisha hitilafu, uboreshaji au vipengele vipya. Fungua [tole](https://help.github.com/articles/creating-an-issue/ 'Mastering Issues | GitHub Guides') au tuma [ombi la kuvuta] (https://help.github.com) /makala/kuunda-ombi-kuvuta-kutoka-uma/ 'Kuunda ombi la kuvuta kutoka kwa uma | Msaada wa GitHub')
 - Ili kusaidia kuboresha jumuiya yetu, angalia kichupo cha GitHub [Majadiliano](https://github.com/Syknapse/Contribute-To-This-Project/discussions) kilicho karibu na Vuta Maombi. Eneo hili ni mahali pa kujitambulisha, kuingia katika majadiliano ya kina kuhusu Chanzo Huria, na kuwasiliana na Wasimamizi wa Mradi. Je, utatusaidia kujenganje ya kipengele hiki na kuboresha jumuiya yetu?
-- **Asante kwa kuchangia mradi huu**. Sasa unaweza kuendelea na kujaribu kuchangia miradi mingine; tafuta ![Toleo zuri la Kwanza](https://user-images.githubusercontent.com/29199184/33852733-e23b7070-debb-11e7-907b-4e7a03aad436.png) lebo kwa chaguo za michango zinazofaa kwa wanaoanza.
+- **Asante kwa kuchangia mradi huu**. Sasa unaweza kuendelea na kujaribu kuchangia miradi mingine; tafuta ![Toleo zuri la Kwanza](https://user-images.githubusercontent.com/29199184/33852733-e23b70 70-debb-11e7-907b-4e7a03aad436.png) lebo kwa chaguo za michango zinazofaa kwa wanaoanza.
```

Comment:
again the links shoudn't be changed in any circumstances. they should be same as original README (eng)